apps/ima/translations/src/en/pages.json

```diff
@@ -136,17 +136,32 @@
       "case-details": {
         "header": "Case details"
       },
-      "human-rights": {
-        "header": "Human rights"
-      },
       "immigration-adviser-details": {
         "header": "Immigration adviser details"
       },
       "personal-details": {
         "header": "Claimant details"
       },
+      "exception-duty-to-remove": {
+        "header": "Exception duty to remove"
```

Comment:
The header should be 'Exception to duty to remove"

Reply:
Changes done